### DISCUSSION TITLE
Random Password Generator

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1761] [Suggestions]
Hello everyone, and hello developer of this App.
I'd like to see a very simple added functionality here: a random password generator.
Nothing too fancy, just something like this (Android) App: https://play.google.com/store/apps/details?id=com.vecturagames.android.app.passwordgenerator

You can understand the App by just looking at the screenshot,, there's really nothing much other than that.

Thanks for reading this request, and thanks much more if you please me.

Regards, have a nice day. =)

**Comment by alexx2000** [comment]
Hi, I don't know how is the file manager related to this feature.

**Comment by anonimo82** [comment]
No, indeed, sorry. I posted the wrong repo, please, excuse me.

**Comment by anonimo82** [comment]
But a thing I'd like to see in DoubleCMD is some Id3 tag editing functionality
